packages/closer/pages/members/[slug].tsx

```
@@ -290,12 +275,16 @@ const MemberPage = ({ member, loadError }) => {
-                              return skill + ', ';
-                            })}
+                            {member.preferences?.skills &&
+                              member.preferences?.skills?.map((skill, i) => {
+                                if (
+                                  i ===
+                                  (member.preferences?.skills?.length || 1) - 1
+                                ) {
+                                  return skill;
+                                }
+                                return skill + ', ';
+                              })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)